README.md

```diff
@@ -1,22 +1,24 @@
 ## BUB2
 A book uploader bot that transfers documents from public libraries such as Google Books, and Panjab Digital Library etc to Internet Archive. Built for Wikimedia Tool Labs.
 
-## Local Setup instructions
-1. Install [Node.js](https://nodejs.org/en/download/)
-2. Install and setup Redis server on your machine.
-3. Clone the repository
-<br />`git clone https://github.com/coderwassananmol/BUB2`
-4. Change the directory:
-<br />`cd BUB2`
-5. Checkout to develop branch:
-<br /> `git checkout develop`
-6. Run <br /> `npm install`
-7. Start redis service in background.
-8. Open `.env.example` in your editor and add the information and rename it to .env.
-9. Run <br /> `npm run dev` for development and `npm run start` for production
-10. Open your browser and run your local server: [http://localhost:3000](http://localhost:3000)
-
-Note: In order to send email through gmail, you may need to allow less secure app access. To turn it on, go to https://myaccount.google.com/lesssecureapps?pli=1
+## Getting Started
+1. Install [Node](https://nodejs.org/en/download/). Check for Node and NPM correct install using `node -v` and `npm -v` on the terminal.
+2. Install [Docker toolbox](https://docs.docker.com/toolbox/toolbox_install_windows/).
```

Comment:
Thanks for the PR!
Can you please mention that apart from docker, you can run the redis as background service as well. Other than that, it is quite good. Just make the change, and I will merge it.